packages/cloud_firestore/cloud_firestore/lib/src/utils/PlatformUtils.dart

```diff
@@ -0,0 +1,28 @@
+part of cloud_firestore;
+
+class PlatformUtils {
+  static platform.Source _toPlatformSource(Source platformSource) {
+    switch (platformSource) {
+      case Source.cache:
+        return platform.Source.cache;
+      case Source.server:
+        return platform.Source.server;
+      case Source.serverAndCache:
+        return platform.Source.serverAndCache;
+      default:
+
```

Comment:
@ditman Can you point me to direction of the styleguide? 

It would be great if you are able to share intelliJ/AndroidStudio editor config xml file

Reply:
For plugins, all you need to do is `dartfmt -w .` in the directory of the plugin, it should apply the default dart styleguide. Note that if you want to touch framework code (flutter/flutter) you don't want to auto-format with dartfmt.

I use vscode and normally do this by hand, however [this may be helpful](https://www.jetbrains.com/help/idea/code-style-dart.html) (?)